graylog2-web-interface/src/components/common/PaginatedList.jsx

```diff
@@ -114,23 +115,29 @@ class PaginatedList extends React.Component<Props, State> {
     const numberPages = Math.ceil(totalItems / pageSize);
 
     return (
-      <>
-        {this._pageSizeSelect()}
-
-        {children}
-
-        <div className="text-center">
-          <Pagination bsSize="small"
-                      items={numberPages}
-                      maxButtons={10}
-                      activePage={currentPage}
-                      onSelect={this._onChangePage}
-                      prev
-                      next
-                      first
-                      last />
-        </div>
-      </>
+      <InteractiveContext.Consumer>
```

Comment:
@dennisoelkers has made a shortcut for that. `<IfInteractive>` 

Take a look here: https://github.com/Graylog2/graylog2-server/blob/master/graylog2-web-interface/src/views/components/widgets/Widget.jsx#L291